util/src/lib.rs

```diff
@@ -93,6 +95,7 @@ pub fn chdir(p: &std::path::Path) {
 }
 
 #[cfg(feature = "github")]
+#[derive(Clone)]
```

Comment:
Fixed by making `update_metadata_comment` take a mutable reference. It is also necessary to do so to save the newly created comment id in the variable so that on the next call to `update_metadata_comment` it will not create a new comment if it has already been created.